### PR TITLE
5 packages from AestheticIntegration/catapult at 0.2

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`catapult.0.2`: Tracing system based on the Catapult/TEF format
-`catapult-client.0.2`: Network client for catapult, to be paired with catapult-daemon
-`catapult-daemon.0.2`: Daemon for reliable multi-process logging with catapult
-`catapult-file.0.2`: File logger for catapult
-`catapult-sqlite.0.2`: Sqlite-based backend for Catapult tracing

---
* Homepage: https://github.com/AestheticIntegration/catapult
* Source repo: git+https://github.com/AestheticIntegration/catapult.git
* Bug tracker: https://github.com/AestheticIntegration/catapult/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/catapult-client/catapult-client.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+bug-reports: "https://github.com/AestheticIntegration/catapult/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "catapult" {= version}
+  "odoc" {with-doc}
+  "zmq" {>= "5.0"}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/catapult-client/catapult-client.0.2/opam:16]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/catapult-daemon/catapult-daemon.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+depends: [
+  "dune" {>= "2.0"}
+  "odoc" {with-doc}
+  "catapult" {= version}
+  "catapult-sqlite" {= version}
+  "zmq" {>= "5.0"}
+  "logs" {>= "0.7"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/catapult-daemon/catapult-daemon.0.2/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/catapult-file/catapult-file.0.2/opam [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+homepage: "https://github.com/AestheticIntegration/catapult"
+bug-reports: "https://github.com/AestheticIntegration/catapult/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "catapult" {= version}
+  "odoc" {with-doc}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/catapult-file/catapult-file.0.2/opam:15]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/catapult-sqlite/catapult-sqlite.0.2/opam [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+depends: [
+  "dune" {>= "2.0"}
+  "sqlite3" {>= "5.0"}
+  "directories"
+  "catapult" {= version}
+  "odoc" {with-doc}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/catapult-sqlite/catapult-sqlite.0.2/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/catapult/catapult.0.2/opam [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+depends: [
+  "dune" {>= "2.0"}
+  "base-threads"
+  "base-unix"
+  "odoc" {with-doc}
+  "ocaml" {>= "4.08"}
+]
+depopts: ["mtime"]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/catapult/catapult.0.2/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Thanks

**Comment by c-cube** [comment]
thank you :)

**Pull request merged**